### PR TITLE
Update docker healthchecks

### PR DESCRIPTION
**Pull request by thomschke**

### CODE REVIEW

File: dockerfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ RUN adduser -D extractor
 WORKDIR /home/extractor
 
 RUN apk add --update --no-cache --virtual .build-deps g++ python3-dev libxml2 libxml2-dev libffi-dev openssl-dev
-RUN apk add libxslt-dev
+RUN apk add libxslt-dev curl
```
Comment (RMeissnerCC) [inline dockerfile:8]: Why curl? I do not see it anywhere else in the project, yet.

Comment (thomschke) [reply dockerfile:8]: the extractor healthcheck is using `curl` https://github.com/codecentric/metadata_picker/blob/4345a4709ddd9f5195af2b5c1ac9f8957fcf5ea9/docker-compose.yml#L12

Comment (RMeissnerCC) [reply dockerfile:8]: Interesting. I had the impression it worked so far. Thanks for adding it.

**Review by RMeissnerCC — COMMENTED** [review]
Done

**Pull request merged**